packages/interaction-ring/src/InteractionRing.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { useUsingKeyboardContext } from '@leafygreen-ui/leafygreen-provider';
+import { uiColors } from '@leafygreen-ui/palette';
+
+const Mode = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Mode = typeof Mode[keyof typeof Mode];
+
+interface ColorSet {
+  interactionRingHover: string;
+  interactionRingFocus: string;
+}
+
+const colorSets: Record<Mode, ColorSet> = {
+  [Mode.Light]: {
+    interactionRingHover: uiColors.gray.light2,
+    interactionRingFocus: '#9dd0e7',
```

Comment:
@DesignerDave we should codify this value somewhere

Reply:
I'm aware – outside the scope of this though.

An aside, I'm still not a fan of the colorSets approach. Makes it much more difficult to see the underlying CSS implementation, which is more important than being able to type this IMO.